.rubocop.yml

```diff
@@ -2,114 +2,139 @@ require:
   - rubocop-performance
   - rubocop-rails
 
-inherit_from:
-  - .rubocop_todo.yml
+inherit_from: .rubocop_todo.yml
 
 AllCops:
+  TargetRubyVersion: 2.5
+  TargetRailsVersion: 5.2
   Exclude:
-    - 'config/routes.rb'
-    - 'node_modules/**'
-    - 'test/**/*'
-    - 'doc/plugins/**'
+    - 'node_modules/**/*'
 
-Rails:
-  Enabled: true
-  Exclude:
-    - 'lib/foreman_remote_execution_core/**'
-    - 'node_modules/**'
+Bundler/OrderedGems:
+  Enabled: false
+
+Layout/ArgumentAlignment:
+  EnforcedStyle: with_fixed_indentation
+  IndentationWidth: 2
```

Comment:
Is this something that's being enabled foreman-wide?

Reply:
This is enabled in core. Basically I copied over the contents of core's rubocop.yml...

Reply:
I'm not very fond of this one, but if let's stick with what core uses

Reply:
I concur with that, but hope after enabling/configuring a few more multi line cops in the future it will become nicer overall. :nail_care: